docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     ports:
       - 4444:4444
     volumes:
-      - $HOME/.msf4:/root/.msf4
+      - /tmp/.msf4:/home/msf/.msf4
```

Comment:
Why are you changing this to tmp? The main purpose of this is to mount the users local .msf4 directory into the container to have custom modules and so on.